{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -22,9 +22,16 @@
 import edx_theme
 
 import django
+from django.conf import settings
 from django.utils import six
 
+REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(REPO_ROOT)
+
+import {{ cookiecutter.app_name }}  # pylint: disable=wrong-import-position
```

Comment:
Instead of trying to import the app here, how about parsing the version from the file the same way [setup.py does](https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/setup.py#L15)?  That function's small enough to just copy here; then you shouldn't need to skip linting this file.